.palantir/revapi.yml

```diff
@@ -43,9 +49,6 @@ acceptedBreaks:
     - code: "java.method.removed"
       old: "method org.apache.iceberg.RowDelta org.apache.iceberg.RowDelta::validateNoConflictingAppends(org.apache.iceberg.expressions.Expression)"
       justification: "Deprecations for 1.0 release"
-    - code: "java.method.addedToInterface"
-      new: "method org.apache.iceberg.TableScan org.apache.iceberg.TableScan::useRef(java.lang.String)"
-      justification: "Adding table scan APIs to support scanning from refs"
```

Comment:
@nastra, do you know why revapi suppressions get reordered? Is this a Java version issue?

Reply:
I've got the same diff by running `./gradlew :iceberg-api:revapiAcceptAllBreaks --justification "ManageSnapshots creation through TX"` so it must be RevAPI re-ordering stuff, because previously some things were manually added there in the wrong order. 
Btw, rebasing the PR fixes this, because in some recent commit we fixed the ordering in the revapi file. 
So after rebasing you get the right diff:
```
diff --git a/.palantir/revapi.yml b/.palantir/revapi.yml
index f21466470..1dbfe00e9 100644
--- a/.palantir/revapi.yml
+++ b/.palantir/revapi.yml
@@ -66,6 +66,10 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.io.DataWriter<T>::add(T)"
       justification: "Removing deprecated method"
   "1.1.0":
+    org.apache.iceberg:iceberg-api:
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.ManageSnapshots org.apache.iceberg.Transaction::manageSnapshots()"
+      justification: "ManageSnapshots creation through TX"
     org.apache.iceberg:iceberg-core:
     - code: "java.method.exception.checkedAdded"
```